doc/nrf/releases/release-notes-1.5.0.rst

```diff
@@ -0,0 +1,1132 @@
+* Development support for nRF5340 in multiprotocol configuration (IEEE 802.15.4 and |BLE|).
+* Added PA/LNA GPIO interface support for RF front-end modules (FEM) in radio driver. The front-end module feature in radio driver currently has support for SKY66112-11 device, but does not support nRF21540 revision 1 or older.
+
+Thread
+------
+
+* Added:
+
+  * Production support for nRF5340 in single protocol configuration.
+  * Development support for nRF5340 in multiprotocol configuration (Thread and |BLE|).
+  * Support for nRF5340 for all samples except the :ref:`ot_ncp_sample` sample.
+  * PA/LNA GPIO interface support for RF front-end modules (FEM) in Thread. The front-end module feature in Thread currently has support for SKY66112-11 device, but does not support nRF21540 revision 1 or older.
+
+* Updated:
+
+  * Optimized ROM and RAM used by Thread samples.
+  * Disabled Hardware Flow Control on the serial port in :ref:`coap_client_sample` and :ref:`coap_server_sample` samples.
+  * Thread 1.1 pre-built libraries:
+
+    * Included the latest OpenThread changes.
+    * Added libraries for nRF5340 platform.
+    * Libraries will be certified after the release in multiple feature variants (variants for each platform yet to be confirmed).
+
+Zigbee
+------
+
+* Added:
+
+  * Supported for production for nRF52833 and nRF52840 ( pending formal certification).
+  * Development support for :ref:`NCP (Network Co-Processor) <ug_zigbee_platform_design_ncp_details>`.
+  * Development support for the nRF5340 DK in single and multi-protocol (Zigbee and |BLE|) configuration for the :ref:`zigbee_light_switch_sample`, :ref:`zigbee_light_bulb_sample`, and :ref:`zigbee_network_coordinator_sample` samples.
+  * PA/LNA GPIO interface support for RF front-end modules (FEM) in Zigbee. The front-end module feature in Zigbee currently has support for SKY66112-11 device, but does not support nRF21540 revision 1 or older.
+  * :ref:`zigbee_ncp_sample` sample, which is a Network Co-Processor example for nRF52840 (DK and dongle) and nRF52833 DK.
+
+	* Supports USB and UART transports.
+	* Enables USB DFU when USB transport is used.
+
+  * New ``zcl ping`` command in the :ref:`lib_zigbee_shell` library.
+  * New libraries that were extracted from common code under :file:`subsys/zigbee/common`:
+
+    * :ref:`lib_zigbee_application_utilities` library
+    * :ref:`lib_zigbee_logger_endpoint` library
+
+* Updated:
+
+  * ZBOSS Zigbee stack to version 3_3_0_6+11_30_2020. See :ref:`zboss_configuration` for detailed information.
+  * Added development (unstable) :ref:`zboss` libraries (v3.4.0.800). See :ref:`zboss_configuration` for detailed information.
+
+nRF Desktop
+-----------
+
+Added selective HID report subscription in the USB state.
+This allows the USB state module to subscribe only for a subset of HID reports.
+If USB does not subscribe for the given HID report, |BLE| HIDs can still receive it.
+
+Common
```

Comment:
Maybe we could add an Edge Impulse subsection here, something like:

Edge Impulse
---------------

* Added :ref:`ei_wrapper` that runs the machine learning model.
* Added samples: :ref:`ei_wrapper_sample` sample and :ref:`ei_data_forwarder_sample`

Reply:
Added .